docs/site/Model.md

```diff
@@ -190,9 +188,198 @@ class Product extends Entity {
+    You can split long descriptions into arrays of strings (lines) to keep line lengths manageable; for example:
+  <pre style="font-size: 75%;">[ "Lorem ipsum dolor sit amet, consectetur adipiscing elit",
+
+"sed do eiusmod tempor incididunt ut labore et dolore",<br> "magna aliqua." ]
+
+</pre>
+    </td></tr>
+
+  <tr>
+  <td><code>excludeBaseProperties</code></td>
+  <td>Array</td>
+  <td><code>['id', 'password']</code></td>
+  <td>
+  Excludes the given list of properties from the base model from being visible. Use this instead of the approach documented under 'Exclude properties from base model' section below.
+  </td>
+  </tr>
+
+  <tr>
+  <td><code>forceId</code></td>
+  <td>Boolean</td>
+  <td><code>true</code></td>
+  <td>
+    If true, prevents clients from setting the auto-generated ID value manually.
+  </td>
+  </tr>
+
+  <tr>
+  <td><code>http</code></td>
+  <td>String</td>
+  <td>None</td>
+  <td>Customized HTTP path for REST endpoints of this model.</td>
+  </tr>
+
+  <tr>
+    <td><code>strict</code></td>
+    <td>Boolean or String</td>
+    <td><code>false</code>.<br/>If the data source is backed by a relational database, then default is  <code>true</code>.</td>
+    <td>
+      Specifies whether the model accepts only predefined properties or not. One of:
+      <ul>
+      <li><code>true</code>: Only properties defined in the model are accepted. Use if you want to ensure the model accepts only predefined properties.
+      If you try to save a model instance with properties that are not predefined, LoopBack throws a ValidationError.
+      </li>
+      <li><code>false</code>: The model is an open model and accepts all properties, including ones not predefined in the model.
+        This mode is useful to store free-form JSON data to a schema-less database such as MongoDB.
+      </li>
+      <li><code>"filter"</code>: Only properties defined in the model are accepted.
+      If you load or save a model instance with properties that are not predefined, LoopBack will ignore them. This is particularly useful when dealing with old data that you wish to lose without a migration script.
+      </li>
+      </ul>
+    </td>
+  </tr>
+
+  <tr>
+    <td><code>idInjection</code></td>
+    <td>Boolean</td>
+    <td><code>true</code></td>
+    <td>
+      Whether to automatically add an <code>id</code> property to the model:
+    <ul>
+    <li><code>true</code>: <code>id</code> property is added to the model automatically. This is the default.</li>
+    <li><code>false</code>: <code>id</code> property is not added to the model</li>
+    </ul>
+    See <a href="#id-properties">ID properties</a> for more information.  The <code>idInjection</code> property in <code>options</code> (if present) takes precedence.
+    </td>
+  </tr>
+
+  <tr>
+    <td><code>name</code></td>
+    <td>String</td>
+    <td>None</td>
+    <td>Name of the model.</td>
+  </tr>
+
+  <tr>
+    <td><code>plural</code></td>
```

Comment:
This is used by strong-remoting to figure out URL path, e.g. model `Product` is exposed at `/products`.

This setting does not apply to LB4.

Reply:
Same as above: [here](https://github.com/strongloop/loopback-datasource-juggler/blob/master/lib/model-builder.js#L129)

Reply:
See my https://github.com/strongloop/loopback-next/pull/3354#discussion_r306321047